README.md

```diff
@@ -180,7 +183,34 @@ App.configurePlugin('phonegap-plugin-push', {
 
 ### Server
 
-For example in `Meteor.startup()` block of main.js
+For example in `Meteor.startup()` block of main.js. *Version 4.0.0 and above* 
+
+```js
+Push.Configure({
+  apn: {
+    cert: Assets.getText('apnDevCert.pem'),
+    key: Assets.getText('apnDevKey.pem'),
+    passphrase: 'xxxxxxxxx',
+    production: true,
+    topic: 'com.your.app.identifier' //required
```

Comment:
@MrSpark2591 It would be really useful if you could give some details on this `topic` key.